### ISSUE TITLE
We need to validate that the UI is accessible before usability testing

### ISSUE THREAD

**Issue by willeybryan** [cds-snc/notification-api#25]
Dependant https://github.com/cds-snc/notifier-api/issues/43

**What?**
- Once we have it up and running we need to validate that the Notify system is WCAG 2.1 Level AA compliant
- Run it through our Accessibility testing software if possible if not, let me know. I will Talk to Julianna about a sweep so we can integrate that feedback into design. 

**Why?**
- If we add stuff to it that isn't we will just end up making more work for ourselves. It should be so let's validate it.

**Comment by willeybryan** [comment]
Julianna has been contacted to do a review
